ops/services/okta-app/main.tf

```diff
@@ -68,3 +69,10 @@ resource "okta_auth_server_scope" "sr_env" {
   metadata_publish = "NO_CLIENTS"
   default          = false
 }
+
+resource "okta_trusted_origin" "sr_trusted_origin" {
+  count  = local.is_preview ? 0 : length(var.trusted_origin_urls)
```

Comment:
Rather than use indexing, can we pass a map instead?

Reply:
And regarding the `count`, can we use `var.trusted_origin_urls` instead (e.g., does it exist, length of 0, etc.)? That should fix the preview problem of not every env having a trusted origin, in that if there's no value we'll just skip the resource.

Reply:
I've reworked this to pass a list of maps. `count` now processes based on the number of maps contained within that list, and I have confirmed it works on empty lists by running a plan on `test`.